modules/getting-started-cli-connecting-a-database.adoc

```diff
@@ -1,46 +1,45 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-connecting-database_{context}"]
-= Connecting to a database
+= Deploying the database application
 
-Deploy and connect a MongoDB database where the `national-parks-app` application stores location information.
-Once you mark the `national-parks-app` application as a backend for the map visualization tool, `parksmap` deployment uses the {product-title} discover mechanism to display the map automatically.
+The following procedure deploys `mongodb-nationalparks`, which is a MongoDB database for holding the national park location information.
+
+// After you mark the `national-parks-app` application as a back end for the map visualization tool, the `parksmap` deployment uses the {product-title} discover mechanism to display the map automatically.
+// TODO: What does this mean that it displays the map automatically? To mean that seems like it's going to open the browser for me or something.
+// And exactly when does the "discover mechanism" kick in?
```

Comment:
@AlexonOliveiraRH I removed the sentence on line 11 because I didn't understand was it was getting at (see my comments on 12 and 13).

Can you let me know if you think that's important to say somewhere? And if so, exactly when is this kicking in? I assume it's talking about labeling the back-end route, but not sure about the rest

Reply:
@bergerhoffer honestly, I wouldn't stick too much to this explanation, since it's more towards the app itself than OpenShift. IMHO, we should explain something when it's important to understand some of the OCP concepts surrounding the specific chapter topic, which is not the case, so I'm fine with removing this part too.

Reply:
Thanks, will remove